web/src/copy_messages.ts

```diff
@@ -108,35 +107,6 @@ function construct_copy_div($div: JQuery, start_id: number, end_id: number): voi
     }
 }
 
-function insert_and_select_div($div: JQuery, selection: Selection): void {
-    $div.css({
-        position: "absolute",
-        left: "-99999px",
-        // Color and background is made according to "light theme"
-        // exclusively here because when copying the content
-        // into, say, Gmail compose box, the styles come along.
-        // This is done to avoid copying the content with dark
-        // background when using the app in dark theme.
-        // We can avoid other custom styles since they are wrapped
-        // inside another parent such as `.message_content`.
```

Comment:
I think removing this CSS is going to break copying from the dark theme and pasting into Gmail, as detailed in the comments here.

Reply:
OK, so not inserting into the DOM avoids that problem, makes sense.